src/js/apps/admin/sidebar/action-sidebar_app.js

```diff
@@ -32,7 +32,7 @@ export default App.extend({
   },
   onDelete() {
     // Fake a destroy for program-flow-actions
-    if (!this.action.get('_program')) {
```

Comment:
I don't think this is correct.  Pretty sure a `program-action` is completely unaware of what `program-flow` it is related to which is ultimately the reason we have to do this in the first place (despite what the documentation may suggest).. if a program action does not have a program relationship then it is part of a flow.  That's the only way for us to know not to delete it directly.

Reply:
So I think I was just going off the comment here...if it's a program-flow-action, it should have the `program-flow` relationship and we can fake destroy it. If it's a program action, it won't have that and we continue on? Of course I did it at EOD on friday so...

Reply:
Yeah I suppose what it means is that we're fake destroying the action so that the program-flow-action can do the destroy which auto-destroys the action on the BE.  If we destroy the action here everything goes :boom:  But if we don't fake it, the action doesn't get removed from the list.